tests/test_common.py

```diff
@@ -127,65 +138,36 @@ def test_transform(grid):
     grid_a.transform(trans)
     grid_b.transform(trans.GetMatrix())
     grid_c.transform(pyvista.trans_from_matrix(trans.GetMatrix()))
-    assert np.allclose(grid_a.points, grid_b.points)
-    assert np.allclose(grid_a.points, grid_c.points)
+    assert np.allclose(grid_a.points, grid_b.points, equal_nan=True)
+    assert np.allclose(grid_a.points, grid_c.points, equal_nan=True)
```

Comment:
Hopefully, there are no NaN values in the points?...

Reply:
This was discussed in the issue above by @imsodin.

Hypothesis just generates a range of inputs of the types and rules you specify, including random and common edge cases.